### DISCUSSION TITLE
How to Specify content_type in Pika Blocking connection

### DISCUSSION THREAD

**Discussion by sathishiitd** [pika/pika#1465] [Q&A] [answered]
Hi ,
I have a C++ application that sends a protobuf message to RabbitMQ.  C++ application is using AMQP protocol.
I have another Python application that consume protobuf message from RabbitMQ using Pika Blockingconnection.

My application is not consuming the protobuf messages properly. My application is consumed only partial message because it  decoded message using "text/plain" content_type.
 
My question is how to set content type to decode binary data.  Can you share a example code.

**Comment by lukebakken** [comment, marked as answer]
Hello, thanks for using Pika.

An internet search will lead to your answer - https://www.google.com/search?q=pika+%22content+type%22
